src/domain/eventSearch/utils.tsx

```diff
@@ -229,7 +246,7 @@ export const getEventSearchVariables = ({
     end,
     include,
     isFree: isFree || undefined,
-    keyword: [...(keyword ? keyword : []), ...mappedCategories],
+    [keywords]: [...(keyword ? keyword : []), ...mappedCategories],
```

Comment:
I think that this naming for the keywords little bit confusing. Naming indicates that it contains keywords but is actually just name for the parameter.

Maybe something like keywordsParamName woul be better? Or maybe you can come up with event better name :)

Reply:
yea, i was thinking to ask about some name ideas, if we useParamName as a postfix, then it would be nice to make it a convention to use this naming for var params.